packages/maskbook/src/components/InjectedComponents/ToolbarAtTwitter.tsx

```diff
@@ -124,45 +124,44 @@ export function ToolbarAtTwitter(props: ToolbarAtTwitterProps) {
     }, [windowWidth])
     //#endregion
 
+    const IconButtonMemo = useMemo(
```

Comment:
There's an animation effect when click button which would keep showing after expand or shrink, so re-render IconButton immediately when isExpand changes.

Reply:
so why not leave it in the code?🤔